### ISSUE TITLE
Is there anywhere to get Test, validation set for dataset?

### ISSUE THREAD

**Issue by LokiXun** [raywzy/Bringing-Old-Films-Back-to-Life#25]
Hi, it such a  amazing work!
I am trying to train the model on another training data, and I am wondering Is there anywhere to get Test, validation set for dataset? I mean the one after added degradations, in Quantitive result in Table 1.
thanks
